<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonWindow</class>
 <widget class="QMainWindow" name="CommonWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Лицензии ПО</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="tableView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="addBtn">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="detBtn">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="updateTable">
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="about">
    <property name="title">
     <string>О программе</string>
    </property>
   </widget>
   <widget class="QMenu" name="logout">
    <property name="title">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QMenu" name="tables">
    <property name="title">
     <string>Таблицы</string>
    </property>
    <addaction name="users"/>
    <addaction name="licensors"/>
    <addaction name="pos"/>
    <addaction name="pks"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Поиск</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="tables"/>
   <addaction name="menu"/>
   <addaction name="about"/>
   <addaction name="logout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="users">
   <property name="text">
    <string>Пользователи</string>
   </property>
  </action>
  <action name="licensors">
   <property name="text">
    <string>Организации</string>
   </property>
  </action>
  <action name="pos">
   <property name="text">
    <string>Программное обеспечение</string>
   </property>
  </action>
  <action name="pks">
   <property name="text">
    <string>Рабочие станции</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Истекшие лицензии</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
